app/services/foreman/wrapped_exception.rb

```diff
@@ -13,7 +13,7 @@ def message
       super unless @wrapped_exception.present?
 
       cls = @wrapped_exception.class.name
-      msg = @wrapped_exception.message.try(:truncate, 90)
+      msg = @wrapped_exception.message
       super + " ([#{cls}]: #{msg})"
     end
   end
```

Comment:
Re-reading the ticket, it looks like that's also Doms concern. I think simply delegating the method is sufficient, @domcleal ?